uniffi_bindgen/src/bindings/mod.rs

```diff
@@ -97,42 +96,17 @@ where
 }
 
 /// Execute the given script via foreign language interpreter/shell.
-pub fn run_script<P1, P2>(
-    out_dir: Option<P1>,
-    script_file: Option<P2>,
-    language: TargetLanguage,
-) -> Result<()>
+pub fn run_script<P1, P2>(out_dir: P1, script_file: P2, language: TargetLanguage) -> Result<()>
 where
     P1: AsRef<Path>,
     P2: AsRef<Path>,
 {
-    let out_dir: Option<&Path> = out_dir.as_ref().map(|v| v.as_ref());
-    let script_file: Option<&Path> = script_file.as_ref().map(|v| v.as_ref());
+    let out_dir = out_dir.as_ref();
+    let script_file = script_file.as_ref();
```

Comment:
These are a lot simpler without all the `Option` stuff, I like it. (I'll probably find myself missing the ability to launch into a shell for a bit of ad-hoc debugging, but we can bring that back later if we need to).